### DISCUSSION TITLE
请问Nginx-Cloudflare的`yml`中，如何添加`--cap-add=NET_ADMIN --cap-add=NET_RAW`指令？

### DISCUSSION THREAD

**Discussion by despire1119** [MikeWang000000/Natter#72] [Q&A] [answered]
`Nginx-Cloudflare`示例的`yml`中使用`-m iptables`会报`iptables`权限不足。
这个问题在之前使用`natter-docker`时通过示例中的`--cap-add=NET_ADMIN --cap-add=NET_RAW`参数规避了这个问题。
所以想请问`--cap-add=NET_ADMIN --cap-add=NET_RAW`指令如何在`Nginx-Cloudflare`的`yml`中添加？

**Comment by MikeWang000000** [comment, marked as answer]
请参照 https://github.com/MikeWang000000/Natter/blob/master/natter-docker/transmission/docker-compose.yml

**Reply by despire1119** [reply]
感谢回复
